app/build.gradle

```diff
@@ -77,7 +77,7 @@ android {
 
     signingConfigs {
         debug {
-            storeFile file("debug.keystore")
+            storeFile file(System.env.HOME + '/.android/debug.keystore')
```

Comment:
**Automatic Debug Signing** instead of using an in home `debug.keystore`,  the build tools (Gradle and the Android SDK) automatically use a debug keystore that is generated and managed for you. This keystore is typically located in ~/.android/debug.keystore (on Linux/Mac), %USERPROFILE%\.android\debug.keystore (on Windows).